### DISCUSSION TITLE
How do I make my app run in a codesandbox container?

### DISCUSSION THREAD

**Discussion by jacksonk-dev** [codesandbox/codesandbox-client#6404] [Q&A] [answered]
I have an app running well locally but it throws errors on codesandbox. Thought perhaps a container could do the trick but I don't know how to use one. Any help would be greatly appreciated.
![Capture](https://user-images.githubusercontent.com/23702308/149609563-c6742f6d-46ab-4887-bb4b-ae593fbba974.PNG)

**Comment by JamesACS** [comment, marked as answer]
Hey @jacksonk-dev ! 

For coding help you may have faster responses on our Discord: https://discord.gg/DV34HPb3

**Reply by jacksonk-dev** [reply]
@JamesACS, thanks. I actually found something helpful after searching the web for a few hours. Got it working. Apparently I just had to have 
```
{
  "template": "node"
} 
```
in my sandbox.config.json file and re-import my project from github.

**Reply by JamesACS** [reply]
Great news!
